research-software-directory/RSD-as-a-service
# 209 improve team member table

# Change team_member table to have same structure as contributor table 

Changes proposed in this pull request:

* Remove the `name_particle` and `name_suffix` columns from the `team_member` table and add the columns `affiliation`, `role` and `orcid`
* Adapt the migration script to these new columns (orcid data is not present in legacy RSD, hence is always `null`)
* Frontend adaptations to use the new database structure

How to test:

* Rebuild the database and frontend:
	* `docker-compose down --volumes && docker-compose build database frontend && docker-compose up --scale scrapers=0`
* Build and run the migration script:
	* `cd data-migration && docker-compose build && docker-compose up`
* Look at a project, e.g. http://localhost/projects/estep or http://localhost/projects/puregome and confirm that you can see affiliations and roles for team members
	
Closes #209

PR Checklist:
